README.md

```diff
@@ -314,6 +314,7 @@ There are certain restrictions regarding what model versions can be used with CA
 For regular releases, the major version must be identical.
 For pre-release versions (with `rc<number>` in the version), the entire version number (major, minor, patch & pre-release) must be identical.
 
+
```

Comment:
Perhaps out of scope, but up in the README.md there's an example section for the output filenames, line 139 to 147. This is not accurate anymore.

Reply:
I would indeed not see it related to this PR, but might as well change it now that we noticed (for consistency maybe we should just do a separate `fix/README` PR in the future so a PR revert or whatever wouldn't also undo unrelated changes?).

Reply:
Perhaps still is still better to do now, because the changes that are made are not accurate (input always has to have the `.tsv.gz` extension... I'd suggest leaving this until we find the time to fix bug #125

Reply:
`capice predict -i ./resources/predict_input.tsv -m ./tests/resources/xgb_booster_poc.pickle.dat -o ./test_output.tsv.gz -f` seems to work just fine 😅

Looked into it and only training has this requirement:
* https://github.com/molgenis/capice/blob/main/src/molgenis/capice/cli/args_handler_explain.py#L20
```
@property
def _extension(self):
    return '.tsv', '.tsv.gz'
```
* https://github.com/molgenis/capice/blob/main/src/molgenis/capice/cli/args_handler_predict.py#L19
```
@property
def _extension(self):
    return '.tsv', '.tsv.gz'
```
* https://github.com/molgenis/capice/blob/main/src/molgenis/capice/cli/args_handler_train.py#L18
```
@property
def _extension(self):
    return '.tsv.gz'
```

Agreed to streamline everything to `return '.tsv', '.tsv.gz'`? Also, seems rather strange that explain has this field set (to these values) as it doesn't even accept a tsv file as input.

Reply:
Fixed. Also found an invalid error that was thrown (`FileNotFoundError` instead of `IOError`), so adjusted this as well.

Reply:
Will fix it in separate PR as quality gate fails due to low coverage.